packages/apps/kadena-docs/src/components/Layout/Full/Full.tsx

```diff
@@ -1,30 +1,55 @@
-import { Content } from '../components';
+import { Heading, styled } from '@kadena/react-components';
 
+import { Article, Content } from '../components';
+
+import { AsideLink } from './AsideLink';
+import { Aside, AsideList } from './style';
 import { useCreateSubMenu } from './useCreateSubmenu';
 
 import { ILayout, ISubHeaderElement } from '@/types/Layout';
-import Link from 'next/link';
 import React, { FC, ReactNode } from 'react';
 
+const StickyAsideWrapper = styled('div', {
+  position: 'fixed',
+  display: 'flex',
+  top: '$20',
+});
+
+const StickyAside = styled('div', {
+  position: 'sticky',
+  top: '40px',
+});
+
 export const Full: FC<ILayout> = ({ children }) => {
-  const { headers } = useCreateSubMenu();
+  const { docRef, headers } = useCreateSubMenu();
 
   const renderListItem = (item: ISubHeaderElement): ReactNode => {
     return (
-      <li key={item.slug}>
-        <Link href={`#${item.slug}`}>{item.title}</Link>
-
+      <AsideLink href={`#${item.slug}`} key={item.slug} label={item.title}>
         {item.children.length > 0 && (
-          <ul>{item.children.map(renderListItem)}</ul>
+          <AsideList inner={true}>
+            {item.children.map(renderListItem)}
+          </AsideList>
         )}
-      </li>
+      </AsideLink>
     );
   };
+
   return (
     <>
       <Content id="maincontent">
-        <ul>{headers.map(renderListItem)}</ul>
-        {children}
+        <Article ref={docRef}>{children}</Article>
+
+        <Aside>
+          <StickyAsideWrapper>
+            <StickyAside>
+              <Heading as="h6" bold={true} transform="uppercase">
```

Comment:
nit: bold is true by default on headings so you can remove this if you want to clean up props